### ISSUE TITLE
HW multisig message signing

### ISSUE THREAD

**Issue by mandelbit** [bitcoin-core/HWI#524]
Am I correct in stating there is currently no standard for signing messages with multisig HW cosigners?

It'd be nice if HWI supported this feature.

**Comment by achow101** [comment]
Correct, there is no standard for signing messages with a multisig. There is, in general, no standard for signing messages with anything other than a p2pkh address. [BIP 322](https://github.com/bitcoin/bips/blob/master/bip-0322.mediawiki) aims to solve that, but it seems like progress has stalled on it.

**Comment by achow101** [comment]
Closing as part of sunsetting described in #850
